MIP102/MIP102c2-Subproposals/MIP102c2-SP20.md

```diff
@@ -9,7 +9,7 @@ Author(s): @Rune
 Contributors:
 General Edit or Article 1 Edit: General Edit
 Tags: 
-Status: RFC
+Status: Formal Submission
 Date Proposed: 2023-10-05
 Date Ratified: <yyyy-mm-dd>
 Forum URL: https://forum.makerdao.com/t/mip102c2-spxx-mip-amendment-subproposal/22334
```

Comment:
Update forum URL to: https://forum.makerdao.com/t/mip102c2-sp20-mip-amendment-subproposal/22334